tests/parm/diag_table/diag_table_hrrr

```diff
@@ -192,7 +192,6 @@
 "gfs_sfc",     "stype",       "sotyp",     "fv3_history2d",  "all",  .false.,  "none",  2
 "gfs_sfc",     "slmsksfc",    "land",      "fv3_history2d",  "all",  .false.,  "none",  2
 "gfs_sfc",     "vfracsfc",    "veg",       "fv3_history2d",  "all",  .false.,  "none",  2
-"gfs_sfc",     "wetness",     "wetness",   "fv3_history2d",  "all",  .false.,  "none",  2
```

Comment:
@SamuelTrahanNOAA Sam, we need this variable in the output to compute fire potential index. Could we revert this change or there is a serious problem with having it in diag_table_hrrr?

Reply:
Also, I made a PR to your repository with changes to diag_table_hrrr, daig_table_hrrr_stoch and diag_table_rap

Reply:
It was inconsistent. I've added it to both hrrr diag tables now.